src/ui/common/src/components/pages/check/id/index.tsx

```diff
@@ -65,6 +68,18 @@ const CheckDetailsPage: React.FC<CheckDetailsPageProps> = ({
     dagId,
   });
 
+  breadcrumbs.push(
+    new BreadcrumbLink(path, node ? node.name : 'Check Details')
+  );
+
+  if (!node) {
+    return (
+      <Layout breadcrumbs={breadcrumbs} user={user}>
```

Comment:
Yeah that totally makes sense. I will add these changes after we merge the refactor to main so that ppl can start developing on top of the refactors